app/controllers/stories_controller.rb

```diff
@@ -122,10 +125,11 @@ def handle_tag_index
     @num_published_articles = if @tag_model.requires_approval?
                                 Article.published.cached_tagged_by_approval_with(@tag).size
                               else
-                                Article.published.cached_tagged_with(@tag).size
+                                Article.published.cached_tagged_with(@tag).where("score > 2").size
                               end
-    number_of_articles = user_signed_in? ? 5 : 45
-    @stories = Articles::Feed.new(number_of_articles: number_of_articles, tag: @tag).published_articles_by_tag
+    @number_of_articles = user_signed_in? ? 5 : SIGNED_OUT_RECORD_COUNT
```

Comment:
Could this be something like 

```ruby
def number_of_articles(signed_in:)
  user_signed_in? ? signed_in : SIGNED_OUT_RECORD_COUNT
end
```

Then this would become

```ruby
@number_of_articles = number_of_articles(signed_in: 5)
```

and line 300 would be 

```ruby
...per(number_of_articles(signed_in: 2))
```
which would also make the magic numbers a bit less magic.